edgemesh/tools/initContainer/createImg.sh

```diff
@@ -3,8 +3,8 @@
 echo 'create edgemesh init Container image'
 
 function usage() {
-	echo "execute 'sh createImg.sh [rpm | deb]' to create docker image"
-	echo "execute 'sh createImg.sh help for use help'"
+	echo "execute 'bash createImg.sh [rpm | deb]' to create docker image"
+	echo "execute 'bash createImg.sh help for use help'"
```

Comment:
What's the difference of them? :)

Reply:
Oh, just read the PR description, small question, does the bash exist on all os?

Reply:
verified on ubuntu18 and ubuntu20, not sure on other OS.

Reply:
but we cannot rely on the how the symlink is created, it's more safe to run with "bash" explicitly.